src/transformers/keras_callbacks.py

```diff
@@ -97,10 +97,11 @@ def __init__(
         self.predict_with_generate = predict_with_generate
         self.output_cols = output_cols
         self.metric_fn_kwargs = metric_fn_kwargs or dict()
-        if tokenizer is not None:
```

Comment:
That was the only use of the `tokenizer`, so let's remove it from the init (PR was merged yesterday so we can break slightly like this :-) )